EKF/ekf_helper.cpp

```diff
@@ -1596,6 +1596,7 @@ void Ekf::calcExtVisRotMat()
 	// calculate the quaternion delta between the EV and EKF reference frames at the EKF fusion time horizon
 	Quatf q_error = _state.quat_nominal * _ev_sample_delayed.quat.inversed();
 	q_error.normalize();
+	q_error.canonicalize();
```

Comment:
This step appears unnecessary. If a canonical form is required for an external accessor, then the operation should be done there.